packages/@vue/cli-shared-utils/lib/logger.js

```diff
@@ -1,4 +1,5 @@
 const chalk = require('chalk')
+const stripAnsi = require('strip-ansi');
```

Comment:
Please remove the extraneous semicolon here, as we use the semicolon-less style in this codebase

Reply:
Thanks for the helpful info, the semicolon is now removed!